config/initializers/mailjet.rb

```diff
@@ -0,0 +1,7 @@
+if Rails.env.production?
+  Mailjet.configure do |config|
+    config.api_key = ENV['MAILJET_API_KEY']
+    config.secret_key = ENV['MAILJET_SECRET_KEY']
```

Comment:
Hey @urfolomeu, has these keys been added to the file that loads this into the production environment? I'm guessing there will be a PR for adding this?